torchmetrics/image/lpip_similarity.py

```diff
@@ -155,3 +157,36 @@ def compute(self) -> Tensor:
             return self.sum_scores / self.total
         if self.reduction == "sum":
             return self.sum_scores
+
+
+class LPIPS(LearnedPerceptualImagePatchSimilarity):
+    """The Learned Perceptual Image Patch Similarity (`LPIPS_`) is used to judge the perceptual similarity between
+    two images. LPIPS essentially computes the similarity between the activations of two image patches for some
+    pre-defined network.
+
+    .. deprecated:: v0.7
+        Use :class:`torchmetrics.image.LearnedPerceptualImagePatchSimilarity`. Will be removed in v0.8.
+
+    Example:
+        >>> import torch
+        >>> _ = torch.manual_seed(123)
+        >>> lpips = LPIPS(net_type='vgg')
+        >>> img1 = torch.rand(10, 3, 100, 100)
+        >>> img2 = torch.rand(10, 3, 100, 100)
+        >>> lpips(img1, img2)
+        tensor(0.3566, grad_fn=<SqueezeBackward0>)
+    """
+
+    @deprecated(
+        target=LearnedPerceptualImagePatchSimilarity, deprecated_in="0.7", remove_in="0.8", stream=_future_warning
```

Comment:
Should there maybe be tests to make sure that the deprecated warnings work?

Reply:
it minds, but as we use the same package in PL and Flash I hope it works fine
I think that @stancld also tested it locally with all his renames...
In fact, you are asking if the deprecation really raises a warning, as there is still doctest to test the correct outputs :rabbit: